src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1458,12 +1486,14 @@ def update_site_configs(cmd, resource_group_name, name, slot=None, number_of_wor
 def delete_app_settings(cmd, resource_group_name, name, setting_names, slot=None):
```

Comment:
The `slogConfigNames` API will not be supported by the backend. But we do not want to just rethrow the backend's error message since we should be able to do everything else in this logic

Reply:
Good point. We might have to just prevent the customer from passing a value for slot_settings for Centauri function apps.